polldaddy-shortcode.php

```diff
@@ -0,0 +1,463 @@
+<?php
+
+if ( function_exists( 'get_option' ) == false )
+	die( "Cheatin' eh?" );
+
+if ( ! class_exists( 'PolldaddyShortcode' ) ) {
+
+/**
+ * Class wrapper for polldaddy shortcodes
+ */
+class PolldaddyShortcode {
+
+	static $add_script = false;
+	static $scripts = false;
+
+	/**
+	 * Add all the actions & resgister the shortcode
+	 */
+	function __construct() {
+		if ( defined( 'GLOBAL_TAGS' ) == false ) {
+			add_shortcode( 'crowdsignal', array( $this, 'polldaddy_shortcode' ) );
```

Comment:
Is adding this all that changed? It is hard to tell because you moved it.

In the future, you should structure that as 2 separate PRs, so you can track changes easier.

Reply:
Not really. I can still move the later commits into a separate PR to be merged into this one - that should make it a little easier to review than going by individual commits.

Reply:
Ok, thanks

Reply:
Moved the additional commits to #14 which is now ready for review.